scripts/add_part.py

```diff
@@ -54,11 +54,11 @@ def query_args(args):
     connect = part_connect.Connections()
     part = part_connect.Parts()
     handling = cm_handling.Handling(session)
-    part_check = handling.get_part_dossier(hpn=args.hpn, rev=args.rev,
+    part_check = handling.get_part_dossier(hpn_list=[args.hpn], rev=args.rev,
                                            at_date=at_date, exact_match=True)
 
     # Check for part
-    if len(part_check.keys() > 0):
+    if len(part_check.keys()) > 0:
```

Comment:
Stylistically, lengths can never be negative so I like writing `if len(part_check.keys()):` for this sort of thing.